.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
     - RUN_E2E_TESTS_LIBRARY=true
     - RUN_E2E_TESTS_NAVIGATION=true
     - RUN_E2E_TESTS_PREFERENCES=true
-    - RUN_E2E_TESTS_PROFILE_FEATURES=true
+    - RUN_E2E_TESTS_PROFILE_FEATURES=false
```

Comment:
Hi @nithusha21, I think you'll have to remove this completely -- Travis still takes up to 2 mins to go through config file for this test slab. See here: https://travis-ci.org/oppia/oppia/jobs/485570557

Reply:
Ah I see. I'll do that.

Reply:
Done!